### PR TITLE
refactor: remove mempool stream PE-1579

### PR DESCRIPTION
**Pull request by jdaev**
Removes cached mempool stream.

### CODE REVIEW

**Review by fedellen — APPROVED** [review]
👍

**Comment by fedellen** [comment]
Whats the status here? We should push this. It'll be a nice performance boost

**Comment by jdaev** [comment]
Sorry about that. I did this on a weekend and forgot about it.

**Review by karlprieb — APPROVED** [review]

**Review by matibat — APPROVED** [review]

**Review by thiagocarvalhodev — DISMISSED** [review]

**Comment by jdaev** [comment]
🦖 👍🏽

File: lib/pages/congestion_warning_wrapper.dart [hunk 1]
```diff
@@ -9,9 +9,17 @@ import 'package:flutter_bloc/flutter_bloc.dart';
...
-          mempoolWarningSizeLimit;
+  late bool warnAboutCongestion;
+
+  try {
+    final mempoolSize =
+        await context.read<ArweaveService>().getMempoolSizeFromArweave();
+    
+   warnAboutCongestion = mempoolSize > mempoolWarningSizeLimit;
+  } catch (e) {
+    warnAboutCongestion = false;
```
Comment (matibat) [inline lib/pages/congestion_warning_wrapper.dart:20]: Hmmm so now we set it to false when the request fails? Shouldn't we retry? :opionion:

**Review by matibat — APPROVED** [review]
Actually I think we are not changing the behavior. That prev comment shouldn't be a blocker

File: lib/pages/congestion_warning_wrapper.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/pages/congestion_warning_wrapper.dart:20]: We could, but It is not critical. Today we have the GraphQLRetry class to perform retries. When implemented a similar class to all requests, we can address it for sure.

Comment (matibat) [reply lib/pages/congestion_warning_wrapper.dart:20]: Yeah, you are good to go!. Thanks @thiagocarvalhodev 😎

**Pull request merged**